### PR TITLE
Added OCaml (on CodeMirror already)

### PR DESCRIPTION
**Pull request by bmazoure**

### CODE REVIEW

**Comment by Denisov21** [comment]
@ArtificialBreeze must sign this license  http://dev.brackets.io/brackets-contributor-license-agreement.html please!

File: src/language/languages.json [hunk 1]
```diff
@@ -293,5 +293,11 @@
         "fileExtensions": ["hs"],
         "blockComment": ["{-", "-}"],
         "lineComment": ["--"]
-    }
+    },
```
Comment (sprintr) [inline src/language/languages.json]: You need to add opening brace (`{`) here.

File: src/language/languages.json [hunk 2]
```diff
@@ -293,5 +293,11 @@
...
+        "ocaml": {
+        "name": "OCaml",
+        "mode": ["ocaml", "text/x-ocaml"],
+        "fileExtensions": ["ml"],
+        "blockComment": ["(*", "*)"]
+    },
```
Comment (sprintr) [inline src/language/languages.json]: You need to remove comma (`,`) from here.

**Comment by abose** [comment]
Thanks @ArtificialBreeze For contributing to brackets.
As a general guideline, brackets core is targetted towards web development, and hence we avoid languages not related to the web in core. Haskell - being an exception as we accidentally merged it in and forgot to rollback :sweat_smile: 
New language support is in the realm of extensions. That can bring in a much more complete support for the language. 

Waiting for other options on this PR.

**Comment by Denisov21** [comment]
+1 @abose

**Comment by abose** [comment]
Closing as this is better done by extensions.

**Pull request closed without merging**